api/lib/stores/block-store.js

```diff
@@ -264,6 +264,53 @@ class BlockStore {
       'operatorAddress'
     )
   }
+
+  async storeNetwork() {
+    try {
+      // prepare network with the format we are going to store it in public/networks
+      const dbNetwork = formatNetworkForDB(this.network)
+      // store network in DB under public/networks
+      this.db.storeNetwork(dbNetwork)
+    } catch (error) {
+      console.error('Failed during store network in DB', error)
+      Sentry.captureException(error)
+    }
+  }
+
+  async updateNetworkInDB() {
+    try {
+      const storedNetwork = await this.db.getNetwork(this.network.id)
+      if (storedNetwork.length > 0) {
+        // getNetwork only returns one network, so it is safe to do storedNetwork[0]
+        // we update everything except the network id and powered
+        this.network.enabled = storedNetwork[0].enabled
```

Comment:
here you use `[0]` every time. I think `getNetwork` should return an object not an array to avoid code duplication

Reply:
true true